tailwind.config.js

```diff
@@ -45,21 +44,25 @@ module.exports = {
       'admin-gray': '#dcdcde',
 
       // Dark Charcoal
-      'dark-charcoal-06': '#f3f2f2',
-      'dark-charcoal-10': '#eae9ea',
-      'dark-charcoal-20': '#d6d4d5',
-      'dark-charcoal-30': '#c1bec0',
-      'dark-charcoal-40': '#aca9ab',
-      'dark-charcoal-50': '#989397',
-      'dark-charcoal-60': '#837d82',
-      'dark-charcoal-70': '#6e686d',
-      'dark-charcoal-80': '#595258',
+      'dark-charcoal': {
```

Comment:
This will break so many other components. We'll need to find/replace before merging this PR.

Reply:
I already did the find-and-replace.